src/components/draw/feature-measurements-helper.ts

```diff
@@ -9,3 +16,26 @@ export function downloadCsv(profileData: PointData[]) {
     return
   }
 }
+
+export function getElevation(coordinate: Coordinate) {
+  const lonlat = transform(coordinate, MAP_CRS, ELEVATION_CRS)
+  return fetch(`${ELEVATION_URL}?lon=${lonlat[0]}&lat=${lonlat[1]}`).then(
+    async response => {
+      const resp = (await response.json()).dhm
+      let text = i18next.t('N/A', { ns: 'client' })
+      if (resp) {
+        text = `${Number(resp).toFixed(2)} m`
+      }
+      return { formattedElevation: text, rawElevation: resp }
```

Comment:
Is there a reason why you return an object with the formatted value and the raw value? Maybe this: `text = '${Number(resp).toFixed(2)} m'` should be in getFormattedElevation function?

Reply:
also don't know if this is relevant but maube we could create a directive for formatting to m, m2 or km?

Reply:
Good point, thanks for the idea! I've added a `format-distance` directive for the formatting logic from the helper. I'll continue to adopt it for the formatting in the utils in #159.